translations.json

```diff
@@ -1968,6 +1973,7 @@
             "#comment": "The placeholder here is the project being updated (i.e. OverlayPlugin or Cactbot)\nThis text is displayed in ACT's restart prompt. ACT already asks the user to restart so we don't have to.",
             "de-DE": "{0} wurde erfolgreich geupdatet.",
             "en": "{0} was updated successfully.",
+            "fr-FR": "{0} a été mis à jour avec succès.',
```

Comment:
Typo.

```suggestion
            "fr-FR": "{0} a été mis à jour avec succès.",
```

Reply:
👍
Sorry, i'm used to translate cactbot :)

Reply:
We need linting for JSON here to detect this issue. It's somewhere on my TODO list after I get the other code analyzer stuff merged.